app/views/pages/get-started/build-your-first-app/sending-email-notifications.liquid

```diff
@@ -0,0 +1,215 @@
+
+#### /app/views/layout/mailer.liquid
+
+{% raw %}
+```liquid
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+
+    <style>
+      h1 {
+        text-align: center;
+      }
+
+      a {
+        color: #1b8816;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Your <em>To do</em> list has an update!</h1>
+
+    {{ content_for_layout }}
+  </body>
+</html>
+```
+{% endraw %}
+
+The `{% raw %}{{ content_for_layout }}{% endraw %}` is a Liquid variable that will be replaced by the actual email content we’ll define in other `.liquid` files.
+
+If you ever styled an email, you know how many limitations the CSS for emails has. One of the rules to follow is that it’s safest to place the styles inline with the help of the `style` attribute. platformOS makes it easy – whatever styles you’ll include in the layout will be automatically converted to inline styles. In the case above – the `<h1>` will become `<h1 style="text-align: center;">`.
+
+For our _To Do_ app, let’s add an email notification when you add a new item on the list. The content of email messages should be kept in `app/emails/` folder. To follow the pattern we established in this tutorial we’re going to create the file as `app/emails/item/created.liquid`:
+
+#### /app/emails/item/created.liquid
+
+{% raw %}
+```liquid
+---
+from: "no-reply@example.com"
+to: "your-email@example.com"
+subject: "New item on the To Do list"
+layout: mailer
+---
+
+A new item has been created: {{ data.title }}
+```
+{% endraw %}
+
+The top-most part of the code, between the dashed lines (`---`) is the page configuration section. Those are the internal information passed to the platform to describe the page behavior or pass additional metadata.
+
+For the email page we need to configure the recipient email address (`to:`), the sender (`from:`) and the `subject` of the message.
+
+We’ve created a new layout file, especially for this occasion, so the 5th line tells the platform to use the file stored in `app/views/layout/mailer.liquid` as a scaffolding for the email content.
+
+The last line is just the content of the email we’re going to send. Since we would like to pass the title of the item added to the list, we are outputting the content of `data.title`. The `data` will be the name of the variable we are going to use in the GraphQL _mutation_ that will trigger sending the email.
+
+
+## Using GraphQL to send an email
+
+To actually send an email through the platform we need to trigger it via GraphQL. To make it as simple as we can, let’s create a new file under `app/graphql/item/email/created.graphql` that will handle sending the email after we create a new item in the database:
+
+
+#### /app/graphql/item/email/created.graphql
+
+```graphql
+mutation email_item_created(
+  $data: HashObject!
+) {
+  email_send(
+    template: { name: "item/created" }
```

Comment:
if we follow the suggestion of having one generic mutation, we pass 'item/created' as an argument